lib/fetch/request.js

```diff
@@ -358,9 +358,13 @@ class Request {
           ac.abort(this.reason)
         }
 
-        if (getEventListeners(signal, 'abort').length >= defaultMaxListeners) {
-          setMaxListeners(100, signal)
-        }
+        // Third-party AbortControllers may not work with these.
+        // See https://github.com/nodejs/undici/pull/1910#issuecomment-1464495619
+        try {
```

Comment:
I wasn't sure if vitest would override EventTarget as well, meaning an instanceof check wouldn't work. This way also shouldn't slow down anyone not overriding globals too so I'd call it a half-shitty solution lol